autoload/neomake.vim

```diff
@@ -248,10 +248,12 @@ function! s:Make(options) abort
         endif
     endif
 
+    let title = join(map(copy(enabled_makers), 'v:val.name'), ', ')
+
     if file_mode
-        lgetexpr ''
+        call setloclist([], 'r', title)
     else
-        cgetexpr ''
+        call setqflist([], 'r', title)
```

Comment:
@nerdrew you shouldn't need to worry about handling this error if you handle both cases as I suggested earlier. But +1 for thinking about backwards compatibility @blueyed!